optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java

```diff
@@ -51,7 +51,7 @@
     }
 
     /**
-     * The source variables (masters) that trigger a change to this shadow variable (slave).
+     * The source variables (main) that trigger a change to this shadow variable (slave).
```

Comment:
Consistency with the [`main` branch](https://github.com/kiegroup/optaplanner/blob/main/optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java):

```suggestion
     * The source variables (leaders) that trigger a change to this shadow variable (follower).
```